Sourcery/Parsing/Parser.swift

```diff
@@ -252,30 +259,37 @@ final class Parser {
         }
     }
 
-    private func processContainedType(_ type: Any, within containingType: Any) {
+    private func processContainedType(_ type: Any, within containing: Any) {
         ///! only Type can contain children
-        guard let containingType = containingType as? Type else {
-            return
-        }
-
-        switch (containingType, type) {
-        case let (_, variable as Variable):
-            containingType.variables += [variable]
-            if !variable.isStatic {
-                if let enumeration = containingType as? Enum,
-                    let updatedRawType = parseEnumRawType(enumeration: enumeration, from: variable) {
-
-                    enumeration.rawType = updatedRawType
+        if let containingType = containing as? Type {
```

Comment:
keep the guard or use if function and return at the end, if else statements are not very readable and requires us to read everything, the sooner we can break flow the easier it is to process